test/organisations_api_test.rb

```diff
@@ -108,7 +108,10 @@ def organisation(slug: "test-department")
   it "fetches an organisation by slug" do
     hmrc = "hm-revenue-customs"
     api_response = organisation(slug: hmrc)
-    api_response["id"] = "www.gov.uk/api/organisations/#{hmrc}"
+    api_response["id"] = Pact.term(
+      generate: "www.gov.uk/api/organisations/#{hmrc}",
+      matcher: /gov\.uk\/api\/organisations\/#{hmrc}$/,
+    )
```

Comment:
Minor comment: I find it a bit strange that we are using two different ways of matching the host in the same file. Maybe we should use the regex here as well for consistency?

Reply:
Good idea. I did consider it, but dismissed it because the previous test value was `"www.gov.uk/api/organisations/#{hmrc}"`, i.e. missing the `http(s)://` prefix. But looking at it again, I don't think it matters too much - it's quite possible the test value was arbitrarily chosen and that the presence of a HTTP prefix doesn't matter.

I've rewritten the commit, moving the `host_agnostic_endpoint_regex` out of the single test it is in so that it can be reused here.